### DISCUSSION TITLE
How to sort or group and hide system fields?

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#7399] [Archive]
Hello.
How to sort/hide fields in directus system tables on edit view? 

It is important to show some custom fields for user in his/her card, but there are so many system fields, that custom ones look unnoticeable.

**Comment by rijkvanzanten** [comment]
There's no way to alter the layout / field configuration of the directus collections. The only way to hide fields from the user detail page would be to remove the read permissions to those particular fields for the given user role

**Reply by IlyaPetrovM** [reply]
Hm, read permissions, that's a good idea)

**Reply by IlyaPetrovM** [reply]
I use 9.0.0.rc90 and I can't hide any system fields in `directus_users` collection for reading. I do not want to turn off App Access obviously.
![image](https://user-images.githubusercontent.com/6277090/129444559-f7352dc2-aa60-4e61-9f29-31b34b53863a.png)

**Reply by rijkvanzanten** [reply]
The fields that are included by default aren't all the fields. They're only the fields that the app requires in order to display/work properly

**Reply by IlyaPetrovM** [reply]
Is there any way to hide `tfa_secret`, `status`, `location` and `title` for "ordinary" users? This is important for my project. 

Can I hide these fields using phpmyadmin in example?

And how to reorder them? May be if I can not hide them completely, is it possible to rearrange these fields, and set my custom fields in the top of item view?

**Reply by rijkvanzanten** [reply]
No, these fields aren't managed in the database and are therefore not user editable

**Comment by rafarc21** [comment]
Is this still not possible to do? Directus_users for example is very cluttered and having the Token field for example being mandatorily visible creates opportunity for users to use our app in unintended ways. I'd love be able to hide that.
